.github/workflows/update-v8.yml

```
@@ -0,0 +1,56 @@
+name: V8 update
+on:
+  schedule:
+    # Run once a week at 00:05 AM UTC on Sunday.
+    - cron: 5 0 * * 0
+  workflow_dispatch:
+
+env:
+  NODE_VERSION: lts/*
+
+permissions:
+  contents: read
+
+jobs:
+  v8-update:
+    if: github.repository == 'nodejs/node'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c  # v3.3.0
+        with:
+          persist-credentials: false
+      - uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8  # v3.3.1
+        with:
+          path: |
+            ~/.update-v8
+            ~/.npm
+        # Install dependencies
+      - name: Install Node.js
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c  # v3.6.0
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      - name: Install node-core-utils
+        run: npm install -g node-core-utils@latest
+      - name: Check and download new V8 version
+        run: |
+            ./tools/dep_updaters/update-v8-patch.sh > temp-output
+            cat temp-output
+            tail -n1 temp-output | grep "NEW_VERSION=" >> "$GITHUB_ENV" || true
+            rm temp-output
+      - name: Generate commit message if not set
+        if: env.COMMIT_MSG == ''
+        run: |
+          echo "COMMIT_MSG=deps: update v8 to ${{ env.NEW_VERSION }}" >> "$GITHUB_ENV"
```

```suggestion
          echo "COMMIT_MSG=deps: patch V8 to ${{ env.NEW_VERSION }}" >> "$GITHUB_ENV"
```

Also I think you can remove this step and directly define the commit message below.